andela/merry-ah-backend
# #162727457 deploy backend application heroku

#### What does this PR do?
Deploy and define base application for heroku

#### Description of Task to be completed?
- add response object
- install express-request-id to add a unique id to all requests
- setup server environment
- add catch all route for malformed urls
- add index route for home
- add api versioning
- install nodemon for development server
- add script to start nodemon

#### How should this be manually tested?
- clone branch 
- run `npm install` to install dependencies
- run `npm start` or `npm run start:dev` in project root folder
- use postman to hit the following endpoints
   `localhost:9000/` should return `Welcome to Authors Haven`, 
   `localhost:9000/api/v1/`,
   `localhost:9000`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/162727457
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
